src/amo/pages/Guides/index.js

```diff
@@ -0,0 +1,214 @@
+import { sanitizeHTML } from 'core/utils';
+import Icon from 'ui/components/Icon';
+import type { AddonType } from 'core/types/addons';
+import type { AppState } from 'amo/store';
+import type { I18nType } from 'core/types/i18n';
+import type { DispatchFunc } from 'core/types/redux';
+import type { ReactRouterMatchType } from 'core/types/router';
+
+import './styles.scss';
+
+type Props = {|
+  match: {
+    ...ReactRouterMatchType,
+    params: {| guidesSlug: string |},
+  },
+|};
+
+type InternalProps = {|
+  ...Props,
+  addons: Array<AddonType>,
+  clientApp: string,
+  dispatch: DispatchFunc,
+  guidesSlug: string,
+  guids: Array<string>,
+  i18n: I18nType,
+  lang: string,
+|};
+
+export const getGuids = (guidesSlug: string): Array<string> | null => {
+  switch (guidesSlug) {
+    case 'privacy':
+      return ['{446900e4-71c2-419f-a6a7-df9c091e268b}'];
+    // The following will be addressed in another issue.
+    // See https://github.com/mozilla/addons-frontend/issues/6744.
+    case 'organize-tabs-and-bookmarks':
+    case 'enhance-your-media-experience':
+    default:
+      return null;
+  }
+};
+
+export const getContent = (guidesSlug: string, i18n: I18nType): Object => {
+  switch (guidesSlug) {
+    case 'privacy':
+      return {
+        title: i18n.gettext('Stay Safe Online'),
+        introText: i18n.gettext(
+          'The web is a wonderful but wild place. Your personal data can be used without your consent, your activities spied upon, and your passwords stolen. Fortunately, extensions can help fortify your online privacy and security.',
+        ),
+        icon: 'stop-hand',
+        sections: [
+          // Bitwarden free password manager
+          {
+            header: i18n.gettext('Create and manage strong passwords'),
+            description: i18n.gettext(
+              'Password managers can help you create secure passwords, store your passwords (safely) in one place, and give you easy access to your login credentials wherever you are.',
+            ),
+            addonCustomText: i18n.gettext(
+              'Fully encrypted password protection. Store your data securely and access logins across devices.',
+            ),
+            exploreMore: i18n.gettext(
+              'Explore more %(linkStart)spassword manager%(linkEnd)s staff picks.',
+            ),
+            exploreUrl: '/collections/mozilla/password-managers/',
+          },
+        ],
+      };
+    // The following will be addressed in another issue.
+    // See https://github.com/mozilla/addons-frontend/issues/6744.
+    case 'organize-tabs-and-bookmarks':
+    case 'enhance-your-media-experience':
+    default:
+      return {};
+  }
+};
+
+export class GuidesBase extends React.Component<InternalProps> {
+  constructor(props: InternalProps) {
+    super(props);
+
+    const { guidesSlug, guids } = this.props;
+
+    this.props.dispatch(
+      fetchGuidesAddons({
+        guids,
+        errorHandlerId: guidesSlug,
```

Comment:
I think you should use `withFixedErrorHandler()` (HOC), which will inject a `errorHandler` in this component (of type `ErrorHandlerType`).